src/open_inwoner/accounts/models.py

```diff
@@ -66,6 +69,13 @@ class User(AbstractBaseUser, PermissionsMixin):
         max_length=15,
         validators=[validate_phone_number],
     )
+    photo = models.ImageField(
+        verbose_name=_("Begeleider photo"),
```

Comment:
Lets just use "Foto" (or "Photo")

I don't mind Dutch and English, but combining this in one label hurts :)